Quote_Manager_CLI_Peter/src/models.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()   
+
+class Quote(Base):
+    __tablename__ = "quotes"
+    
+    id = Column(Integer, primary_key=True, index=True, autoincrement=True)
+    category = Column(String)
+    author = Column(String)
+    quote = Column(String)
```

Comment:
Your table doesn't have `created_at` and `updated_at` fields, I've mentioned this in the past